Restored Python 3.8 compatibility.